indico/web/flask/util.py

```diff
@@ -276,6 +278,9 @@ def send_file(name, path_or_fd, mimetype, last_modified=None, no_cache=True, inl
         raise NotFound(f'File not found: {path_or_fd}')
     if safe:
         rv.headers.add('Content-Security-Policy', "script-src 'self'; object-src 'self'")
+    if not no_cache:
+        del rv.cache_control.no_cache
+        rv.cache_control.max_age = max_age
```

Comment:
I would get rid of this, and pass it to the flask/werkzeug `_send_file()` call. Maybe adding a check to ensure we only do so with `no_cache=False`?